README.adoc

```diff
@@ -72,6 +72,13 @@ $ mvn clean package
 $ java -jar spring-cloud-dataflow-admin-kubernetes/target/spring-cloud-dataflow-admin-kubernetes-1.0.0.BUILD-SNAPSHOT.jar
 ```
 +
+If any of the default module deployer properties need to be overridden then those properties can be passed as program arguments here.
+For example, if the memory, cpu for a Pod need to be changed from the defaults, then we can override as below. Also, sincethe module deployer uses Spring Cloud Stream's Module Launcher, we can pass module launcher properties as well:
++
+```
+$ java -jar spring-cloud-dataflow-admin-kubernetes/target/spring-cloud-dataflow-admin-kubernetes-1.0.0.BUILD-SNAPSHOT.jar --kubernetes.memory=1024Mi --kubernetes.cpu=500m --kubernetes.launcherProperties.remoteRepositories=repo1,repo2
```

Comment:
I think we should first change the way the k8s admin works, so that it leverages https://github.com/spring-cloud/spring-cloud-dataflow/blob/36e97d667d9ec04be4e9dcb6f8a28ac60a0eab0d/spring-cloud-dataflow-admin-starter/src/main/java/org/springframework/cloud/dataflow/admin/config/AdminProperties.java, similar to what is done [here](https://github.com/spring-cloud/spring-cloud-dataflow-admin-cloudfoundry/blob/727e55bc4aed6e747dbc75b6b251dea0924efdbb/spring-cloud-dataflow-admin-cloudfoundry/src/main/java/org/springframework/cloud/dataflow/module/deployer/cloudfoundry/ApplicationModuleDeployer.java#L251) for CF